tools/releaseBuild/azureDevOps/templates/upload-final-results.yml

```diff
@@ -0,0 +1,14 @@
+parameters:
+  artifactPath:
+  artifactFilter: '*'
+  condition: succeeded()
```

Comment:
Do we want `succeeded()` or `failOrSucceeded()`? Maybe we should be able to upload whatever artifacts were built.

Reply:
Not in a release build, we don't want to upload bad artifacts by default.  It will make retrying impossible.

Reply:
We should think about retry scenarios.  We don't want to leave this folder potentially polluted.